tools/common/Application.cs

```diff
@@ -467,12 +467,12 @@ public void RunRegistrar ()
 			if (Registrar != RegistrarMode.Static)
 				throw new PlatformException (67, "Invalid registrar: {0}", Registrar); // this is only called during our own build
 
-			if (RootAssemblies.Count != 1)
+			if (RootAssemblies.Count < 1)
 				throw ErrorHelper.CreateError (8, "You should provide one root assembly only, found {0} assemblies: '{1}'", RootAssemblies.Count, string.Join ("', '", RootAssemblies.ToArray ()));
```

Comment:
Need to update the error message to something like `No assemblies were passed to mmp. You should provide at least one assembly.`

Reply:
And if you do that you will need to update: https://github.com/xamarin/xamarin-macios/blob/master/docs/website/mmp-errors.md (;

Reply:
Or maybe we just add a new error code and error message and obsolete the old one. Are error messages/codes public API, and how closely do you want to maintain semantics in them?